### PR TITLE
Refactoring config load functions

### PR DESCRIPTION
**Pull request by yukirii**
**What this PR does / why we need it**:

* Refactored code for loading credentials
* Improved error messages

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:

Fixes #[PUT ISSUE NUMBER HERE]

**Special notes for your reviewer** *(optional)*:

### CODE REVIEW

File: cmd/root.go [hunk 1]
```diff
@@ -57,69 +57,57 @@ func initConfig() {
...
-			}
-		} else {
-			for _, o := range DC.Organizations {
-				if o.Name == DC.Current {
-					viper.SetDefault("api_key", o.APIKey)
-					viper.SetDefault("app_key", o.APPKey)
-				}
+		viper.SetDefault("api_key", dogrc.Section("Connection").Key("apikey").String())
+		viper.SetDefault("app_key", dogrc.Section("Connection").Key("appkey").String())
+	} else {
```
Comment (toVersus) [inline cmd/root.go]: How about removing an else statement in terms of the early return?

File: cmd/root.go [hunk 2]
```diff
@@ -57,69 +57,57 @@ func initConfig() {
 	// Check args
 	if len(os.Args) == 1 {
 		return
-	} else if len(os.Args) != 2 {
+	}
+
+	if len(os.Args) == 2 {
```
Comment (toVersus) [inline cmd/root.go]: I'm not sure why we put this validation 🤔

Comment (yukirii) [reply cmd/root.go]: It is need to run these commands without Datadog credentials.
If remove this, user can't see help message or set new credentials:
```shell
% ./dogleash help
2019/08/02 16:21:25
Dogleash config file does not exist. [/Users/user/.config/dogleash/config.yml]

% ./dogleash config set
2019/08/02 16:21:30
Dogleash config file does not exist. [/Users/user/.config/dogleash/config.yml]
```

Comment (toVersus) [reply cmd/root.go]: Ah, I see. Thanks!

File: cmd/root.go (hunk 1 above)
Comment (yukirii) [reply cmd/root.go]: After loading configs from the file, it reads environment variables to overwrite configs.
I think we need to use both if and else statements in this case.

**Pull request merged**